src/components/InfoBar/index.tsx

```diff
@@ -50,22 +50,22 @@ export default function InfoBar(props: Props) {
     <div>
       <AppBarFiller />
       <Title>Status</Title>
-      <Data>{info?.connectivityStatus}</Data>
+      <Data>{info.data?.connectivityStatus}</Data>
```

Comment:
This will re-render whenever is fetching changes even when the data does not change.